pyteal/ast/asset.py

```diff
@@ -48,9 +50,9 @@ def total(cls, asset: Expr) -> MaybeValue:
             asset: An index into Txn.ForeignAssets that corresponds to the asset to check. Must
```

Comment:
The docstring should also say that the `asset` arg can be the asset ID starting in v4.

Reply:
docstring updated.